.github/workflows/awslambda-build.yml

```diff
@@ -0,0 +1,13 @@
+name: 'AWS Lambda Build'
+on:
+  push:
+    paths:
+    - ${{ github.workspace }}/packages/serverless/**
+
+jobs:
+  job_build:
```

Comment:
We are actually missing the upload of the artifact `${{ github.workspace }}/packages/serverless/dist-awslambda-layer/*.zip`